lib/travis/build/appliances/update_apt_keys.rb

```diff
@@ -5,12 +5,24 @@ module Build
     module Appliances
       class UpdateAptKeys < Base
         def apply
-          command = <<-EOF
-          if command -v apt-get &>/dev/null && [[ -d /var/lib/apt/lists ]]; then
-            LANG=C apt-key list | awk -F'[ /]+' '/expired:/{printf "apt-key adv --recv-keys --keyserver keys.gnupg.net %s\\n", $3}' | sudo sh &>/dev/null
-          fi
-          EOF
-          sh.cmd command, echo: false
+          sh.if '"$TRAVIS_OS_NAME" == linux' do
+            command = <<~KEYUPDATE
+            apt-key adv --list-public-keys --with-fingerprint --with-colons \\
```

Comment:
pipe at the end of this line and no newline escape, per shfmt/shellcheck :grin: &lt;/nitpick&gt;